<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>A2Pattern</class>
 <widget class="QMainWindow" name="A2Pattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A2Pattern</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Center Coordinates:         x:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditX">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>710</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelY">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>710</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditY">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>710</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelR">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>710</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius        r:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditR">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>710</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="drawPushButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>710</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>710</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
